Tests/Command/ImportTranslationsCommandTest.php

```diff
@@ -6,7 +6,7 @@
 use Doctrine\ORM\EntityManager;
 use Doctrine\ORM\Tools\Console\Command\SchemaTool\CreateCommand;
 use Doctrine\ORM\Tools\Console\Command\SchemaTool\DropCommand;
-use Doctrine\ORM\Tools\Console\EntityManagerProvider;
+// Removed unused use directive
```

Comment:
This comment can go @shakaran

Reply:
Done at https://github.com/lexik/LexikTranslationBundle/pull/472/commits/cdc55891082b9404de35d55e66932fe77a6ee2c0